app/helpers/application_helper.rb

```diff
@@ -24,6 +24,10 @@ def yml_key
     end
   end
 
+  def active_nav_item?(test_path)
```

Comment:
I feel like the argument variable should be called differently to be clearer on what that is